gsy_myco_sdk/utils.py

```diff
@@ -15,3 +17,37 @@ def websocket_domain_name_from_env():
 
 def simulation_id_from_env():
     return os.environ.get("MYCO_CLIENT_SIMULATION_ID", MYCO_CLIENT_SIMULATION_ID)
+
+
+def log_recommendations_response(data: dict) -> None:
```

Comment:
The myco user has to implement this into his matcher in order to get activated, right?

Reply:
I think we decided to use the types from `typing` 
```suggestion
def log_recommendations_response(data: Dict) -> None:
```

Reply:
> The myco user has to implement this into his matcher in order to get activated, right?